presto-main/src/main/java/com/facebook/presto/server/PluginManagerConfig.java

```diff
@@ -114,4 +115,17 @@ public PluginManagerConfig setDisabledConnectors(String disabledConnectors)
         this.disabledConnectors = ImmutableSet.copyOf(Splitter.on(',').omitEmptyStrings().trimResults().split(disabledConnectors));
         return this;
     }
+
+    @NotNull
+    public String getDefaultNamespacePrefix()
+    {
+        return defaultNamespacePrefix;
+    }
+
+    @Config("presto.default-namespace")
+    public PluginManagerConfig setDefaultNamespacePrefix(String defaultNamespacePrefix)
```

Comment:
Rather than match a default namespace with a namespace that is configured in the function namespace manager, just make it a boolean flag property on the function namespace manager's config that it is the default namespace.  If none have that specified, use `presto.default`, if more than one have that specified, then throw.  

The place to add this property would be in `StaticFunctionNamespaceStore`.